apps/admin-portal/src/components/certificates/advanced-search-certificates-keystore.tsx

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Divider, Form, Grid } from "semantic-ui-react";
+import { Field, Forms } from "@wso2is/forms";
+import React, { FunctionComponent, useState } from "react";
+import { AdvancedSearch } from "@wso2is/react-components";
+import { AdvancedSearchIcons } from "../../configs";
+import { SearchUtils } from "@wso2is/core/utils";
+import { useTranslation } from "react-i18next";
+
+/**
+ * Filter attribute field identifier.
+ * @type {string}
+ */
+const FILTER_ATTRIBUTE_FIELD_IDENTIFIER = "filerAttribute";
+
+/**
+ * Filter condition field identifier.
+ * @type {string}
+ */
+const FILTER_CONDITION_FIELD_IDENTIFIER = "filerCondition";
+
+/**
+ * Filter value field identifier.
+ * @type {string}
+ */
+const FILTER_VALUES_FIELD_IDENTIFIER = "filerValues";
+
+/**
+ * The default search strategy. Search input will append the text
+ * field value to this.
+ * @type {string}
+ */
+const DEFAULT_SEARCH_STRATEGY = "alias co";
+
+/**
+ * Prop types for the application search component.
+ */
+interface CertificatesKeystoreSearchPropsInterface {
+    onFilter: (query: string) => void;
+}
+
+/**
+ * Application search component.
+ *
+ * @return {JSX.Element}
+ */
+export const CertificatesKeystoreSearch: FunctionComponent<CertificatesKeystoreSearchPropsInterface> = (
+    props: CertificatesKeystoreSearchPropsInterface
+): JSX.Element => {
```

Comment:
Fix in doc comment and other places.

```suggestion
): ReactElement => {
```